gnocchi/storage/__init__.py

```diff
@@ -514,7 +514,7 @@ def _compute_and_store_timeseries(self, metric, measures):
             LOG.debug("Skipping %s (already processed)", metric)
             return
 
-        measures.sort(order='timestamps')
+        measures = numpy.sort(measures, order='timestamps')
```

Comment:
why'd you change this to create new array vs inplace?

Reply:
Because the array is readonly.

Reply:
ok